feg/gateway/docker/.env

```diff
@@ -10,9 +10,7 @@
 # limitations under the License.
 
 COMPOSE_PROJECT_NAME=feg
-DOCKER_REGISTRY=feg_
-DOCKER_USERNAME=
-DOCKER_PASSWORD=
+DOCKER_REGISTRY=feg-test.artifactory.magmacore.org/
```

Comment:
same

Reply:
From offline: as indicated, I'm not quite sure why we're changing this, but I probably am just misunderstanding, so I'm fine with going for it